lib/graphql/batch.rb

```diff
@@ -19,6 +19,15 @@ def self.batch
       end
     end
 
+    def self.use(schema_defn)
+      if schema_defn.target.class != GraphQL::Schema
```

Comment:
`target` and `use` were added in the same PR upstream (https://github.com/rmosolgo/graphql-ruby/pull/565) so using it here would prevent us from being able to recommend using `GraphQL::Batch.use(self)` for older versions of graphql-ruby.

If you do want to keep this check, then I would change it to `if Gem::Version.new(GraphQL::VERSION) >= Gem::Version.new('1.5.0') && schema_defn.target.class != GraphQL::Schema`